docs/howto_bridge.md

```diff
@@ -5,123 +5,104 @@ title: Bridge Assets
 :::caution
-Note that the only asset(s) currently able to be bridged to Acala from Ethereum is: 
-* ETH: Native
-* DAI: 0x54A37A01cD75B616D63E0ab665bFfdb0143c52AE (always double check contract address)
+Make sure to have enough tokens (ETH and ACA) in your wallets to pay for fees. Remember that fees will be charged for sending and redeeming tokens, and for binding your wallet addresses.
 :::
 
-* Enter the amount of tokens you would like to bridge **(4)**;
-* Click **Next (5)**. This will prompt Metamask to request a network switch from Acala to Ethereum.
+### 01 Navigate to Acala Bridge Page
 
-<div style={{textAlign: 'center'}}>
-  <img alt="metadata" src={useBaseUrl('/howto_bridge/outbound_2.jpg')} width="800px" />
-</div>
+https://apps.acala.network/bridge
 
-### 04 Select Payment for Gas
-* Following the network switch to Ethereum, select the payment method for gas **(6).** Note that Manual Payment is the only option bridging from Acala to Ethereum;
-* Click **Next** **(7)** to proceed. This will prompt Metamask to request a network switch back to Acala from Ethereum.
+Once you have been directed to Acala bridge page, follow the actions below:
 
 <div style={{textAlign: 'center'}}>
-  <img alt="metadata" src={useBaseUrl('/howto_bridge/outbound_3.jpg')} width="800px" />
+  <img alt="metadata" src={useBaseUrl('/howto_bridge/aca1.jpg')} width="600px" />
 </div>
 
-### 05 Bridge Tokens
-* Following the network switch back to Acala, click **Approve** **(8)** to proceed. This will prompt a  first transaction within your Metamask wallet to approve the tokens to be transferred;
-* Once this has been executed, click **Transfer** **(8)** (which will appear in place of Approve). This will prompt a second transaction within your Metamask wallet to execute the bridge transfer.
+### Step 2: Connect to Your Account
 
-<div style={{textAlign: 'center'}}>
-  <img alt="metadata" src={useBaseUrl('/howto_bridge/outbound_4.jpg')} width="800px" />
-</div>
+- Connect your account **(1)**.
+- Select the chains you intend to bridge to and from **(2)**, in this case, it will be **Ethereum** as the Origin Chain and **HydraDX** as the Destination Chain.
+- Connect to your Metamask account that you are bridging from **(3)**.
 
-Once the transaction has been processed by Wormhole, click Redeem (9). This action results in you receiving the tokens on the Ethereum chain.
+### Step 3: Bridge Tokens
 
-<div style={{textAlign: 'center'}}>
-  <img alt="metadata" src={useBaseUrl('/howto_bridge/outbound_5.jpg')} width="800px" />
-</div>
+- Enter the amount of tokens and the token for bridging **(4)**.
+- To commence the bridge, click **Approve Tokens (5)** and sign the transaction using your Metamask wallet app.
+- Once the tokens are approved for transfer, click **Send Tokens (5). T**his starts the bridging process cross-chain.
+- Once the transaction has been processed by Wormhole, click **Redeem Tokens (5).** This action results in you receiving the tokens on the Acala chain.
```

Comment:
You still have to "redeem" the tokens? This doesn't use an auto-relayer? What does the user do once they're on the Acala chain? The UI should abstract the next step out for them to appear on HydraDX?

Reply:
Okay it seems it should be "Redeem & route tokens" - this action results in you receiving the tokens on the HydraDX chain.